lightkurve/lightcurve.py

```diff
@@ -541,33 +542,48 @@ def __repr__(self):
         elif self.mission.lower() == 'k2':
             return('KeplerLightCurveFile(EPIC: {})'.format(self.keplerid))
 
-    def correct(self, method='sff', **kwargs):
+    def correct(self, method='sff', warn=False, **kwargs):
```

Comment:
I'd call this argument `verbose` instead of `warn`.  But it doesn't seem appealing to put a `verbose` parameter in all functions, so I think we shouldn't have this argument and just suppress the polyfit warnings if we are confident that they are useless.

Reply:
I think the polyfit warns will always be useless but I am worried that there may be other warnings from SFFcorrector we may want to see